src/gui/editors/SongEditor.cpp

```diff
@@ -598,10 +600,19 @@ void SongEditor::updatePosition( const MidiTime & _t )
 void SongEditor::zoomingChanged()
 {
 	const QString & zfac = m_zoomingModel->currentText();
-	setPixelsPerTact( zfac.left( zfac.length() - 1 ).toInt() *
-					DEFAULT_PIXELS_PER_TACT / 100 );
+
+	QString zoom = zfac.left( zfac.length() - 1 );
+	if( ! QString::compare("12,5", zoom ) )
+	{
+		zoom = "13";
+	}
+	int pixelsPerTact = zoom.toInt() * DEFAULT_PIXELS_PER_TACT / 100;
+
+	assert( pixelsPerTact > 0 );
```

Comment:
I added the assert just to mirror the PianoRoll.cpp and AutomationEditor.cpp .
As the only variable is the zooming factor I think there is no risk of reaching 0 unless you go down to ~5% zoom.